### PR TITLE
@damassi => Making artwork filter on the artist page dynamic

### PR DESCRIPTION
**Pull request by mzikherman**
Caveat- still has that flashing/whole filter re-rendering, will try to figure that out next week!

bp

It's definitely a rough pass (unselecting a category wasn't working, just selecting another category- probably b/c of the way I was passing the handler to `Radio` component).

Anyway, probably mergeable, and we're getting there!

### CODE REVIEW

File: src/Styleguide/Pages/Artist/Routes/Overview/index.tsx [hunk 1]
```diff
@@ -93,7 +94,7 @@ export const Overview = () => {
 
       <Row>
         <Col>
-          <ArtworkFilter />
+          <ArtworkFilterQueryRenderer artistID="pablo-picasso" />
```
Comment (mzikherman) [inline src/Styleguide/Pages/Artist/Routes/Overview/index.tsx]: Static page is slowly becoming picasso's page 🎉

**Comment by mzikherman** [comment]
Oh yea, we can also get rid of the entire old `Components/ArtistFilter` directory now, that's been migrated here. That'll make the diff have less (+) lines overall.

File: src/Styleguide/Elements/Checkbox.tsx [hunk 2]
```diff
@@ -49,7 +50,13 @@ export class Checkbox extends React.Component<CheckboxProps, CheckboxState> {
     const { children } = this.props
 
     return (
-      <Flex my={0.5} onClick={this.toggleSelected}>
+      <Flex
+        my={0.5}
+        onClick={() => {
+          this.props.onSelect && this.props.onSelect(this.state.selected)
+          this.toggleSelected
+        }}
```
Comment (damassi) [inline src/Styleguide/Elements/Checkbox.tsx]: In a follow up PR we can consolidate / clean up and move this stuff into a handler on the class.

File: src/Styleguide/Pages/Artist/Routes/Overview/index.tsx (hunk 1 above)
Comment (damassi) [reply src/Styleguide/Pages/Artist/Routes/Overview/index.tsx]: screen shot 2018-06-23 at 1 05 20 pm

**Comment by damassi** [comment]
Lets pair on Monday regarding the full-page re-flash as well as QueryRenderer use, but this is a really good first pass 👍 Its coming along nicely!

**Pull request merged**